lib/Agrammon/Web/Routes.pm6

```diff
@@ -185,8 +185,8 @@ sub user-routes(Agrammon::Web::Service $ws) {
 
         # implement/test
         post -> LoggedIn $user, 'change_password' {
-            request-body -> (:$oldPassword!, :$newPassword!) {
-                my $data = $ws.change-password($oldPassword, $newPassword);
+            request-body -> (:$old-password!, :$new-password!) {
```

Comment:
You're fine with the JSON object we get having to have the hyphens in them? (If not, you can do `:oldPassword($old-password)` to use a Raku-style variable name but extract it from another key in the JSON).

Reply:
I guess I overdid the Rakuization :-)